docs/coding-guidelines/updating-ref-source.md

```diff
@@ -15,3 +15,11 @@ These steps can also be applied to some unique assemblies which depend on change
 
 1) Run `dotnet build --no-incremental /t:GenerateReferenceSource` from the System.Runtime/src directory.
 2) Filter out all unrelated changes and extract the changes you care about (ignore certain attributes being removed). Generally, this step is not required for other reference assemblies.
+
+## For Full Facade Assemblies
+
+For assemblies that are "full facades" over another assembly (ex. System.Runtime.Serialization.Json or System.Xml.XDocument), use the following command to generate the reference source code instead:
+
+```
+dotnet msbuild /t:GenerateReferenceAssemblySource /p:GenAPIAdditionalParameters=--follow-type-forwards
```

Comment:
Looks good, maybe we can add a property: `GenAPIFollowTypeForwards` that does this.